CMake Error at cube/CMakeLists.txt:237 (target_link_libraries):
  Target "vkcube" links to:

    volk::volk_headers

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

--- CMakeLists.txt.orig	2024-07-12 20:26:06 UTC
+++ CMakeLists.txt
@@ -56,7 +56,10 @@ find_package(VulkanHeaders QUIET REQUIRED CONFIG)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 find_package(VulkanHeaders QUIET REQUIRED CONFIG)
-find_package(volk QUIET REQUIRED CONFIG)
+find_package(volk QUIET CONFIG)
+if (NOT TARGET volk::volk_headers)
+    add_subdirectory(volk EXCLUDE_FROM_ALL SYSTEM)
+endif()
 if ((APPLE OR BUILD_TESTS) AND NOT ANDROID)
     find_package(VulkanLoader QUIET REQUIRED CONFIG)
 endif()
